README.md

```diff
@@ -57,16 +57,28 @@ ConstellixProvider supports dynamic records.
 
 #### Health Check Options
 
-See https://github.com/octodns/octodns/blob/master/docs/dynamic_records.md#health-checks for information on health checking for dynamic records. ConstellixProvider supports the following options:
+See https://github.com/octodns/octodns/blob/master/docs/dynamic_records.md#health-checks for information on health checking for dynamic records.
 
-| Key  | Description | Default |
-|--|--|--|
-| sonar_interval | Sonar check interval | ONEMINUTE |
-| sonar_port | Sonar check port | 80 |
-| sonar_regions | Sonar check regions for a check. WORLD or a list of values | WORLD |
-| sonar_type | Sonar check type TCP/HTTP | TCP |
+ConstellixProvider supports the following options:
 
-Sonar check interval (sonar_interval) possible values:
+| Key | Description | Default | Notes |
+|--|--|--|--|
+| frequency | time between checks in seconds | 60 | rounded up to the next larger possible sonar_interval |
+| host | FQDN for host header and SNI in HTTP(S) mode | - | ignored for TCP checks |
+| path | path to check in HTTP(S) mode | _dns | ignored for TCP checks |
+| port | port to check | 443 | - |
+| protocol | HTTP/HTTPS/TCP | HTTPS | - |
+
+ConstellixProvider supports the following provider-specific options:
+
+| Key | Description | Default | Notes |
+|--|--|--|--|
+| sonar_interval | Sonar check interval | ONEMINUTE | Deprecated. Use `healthcheck/frequency` instead |
+| sonar_port | Sonar check port | 443 | Deprecated. Use `healthcheck/port` instead |
+| sonar_regions | Sonar check regions for a check. WORLD or a list of values | WORLD | - |
+| sonar_type | Sonar check type TCP/HTTP | HTTPS | Deprecated. Use `healthcheck/protocol` instead |
```

Comment:
Default changed to be in line with
https://github.com/octodns/octodns/blob/main/docs/dynamic_records.md